docs/stencil-docs/storefront-customization/custom-templates.mdx

```diff
@@ -7,7 +7,7 @@ The Stencil framework allows theme developers and merchants to assign custom lay
 * Product
 * Page
 
-This section explains how to author a custom template, map it to a URL, upload it to BigCommerce, and apply it to a storefront as a theme.
+This section explains how to author a custom template, map it to a URL, upload it to BigCommerce, and apply it to a storefront as a theme using Stencil CLI. Mapping a template to a URL is specifically for local testing.
```

Comment:
```suggestion
This section explains how to: author a custom template, map it to a URL, upload it to BigCommerce, and apply it to a storefront as a theme using Stencil CLI. Mapping a template to a URL is specifically for local testing.
```

Reply:
@bc-tgomez I would like to keep this as is because the text includes a sentence and bullet points.